compiler/rustc_lint/src/builtin.rs

```diff
@@ -332,7 +332,7 @@ impl UnsafeCode {
         self.report_unsafe(cx, span, |lint| {
             lint.build(msg)
                 .note(
-                    "the linker's behavior with multiple libraries exporting duplicate symbol \
+                    "the linker behavior with multiple libraries exporting duplicate symbol \
```

Comment:
Is this a typo?

Reply:
it feels incorrect grammatically, but i'm not an expert. Spellchecker didn't like it either.  Not sure

Reply:
it still makes sense grammatically though in this case